generator/src/main.rs

```diff
@@ -1518,7 +1518,11 @@ impl Parser {
                             #(#ext_set_inits)*
                             bytes => {
                                 if let Ok(name) = std::str::from_utf8(bytes) {
-                                    out.other.push(name.into());
+                                    if bytes.last() == Some(&0) {
```

Comment:
Isn't this guaranteed to always be true?

Reply:
It might be. I just didn't want to depend on it. Do you think I should assume it's true and remove the test?

Reply:
I don't mind defensive programming, but it can be very confusing when an interface is used in a way inconsistent with its specified behavior. At least include an explanatory comment?